src/pages/auth/register.tsx

```diff
@@ -38,12 +41,13 @@ const initialSignUpFormValues: ISignUpFormValues = {
 };
 
 // TODO: account types are currently disabled and commented out
-const Register: NextPage = ({}: InferGetServerSidePropsType<
-  typeof getServerSideProps
->) => {
-  const [formValues, setFormValues] = useState<ISignUpFormValues>(
-    initialSignUpFormValues
-  );
+const Register: NextPage = ({
+  csrfToken,
+}: InferGetServerSidePropsType<typeof getServerSideProps>) => {
+  const [formValues, setFormValues] = useState<ISignUpFormValues>({
+    ...initialSignUpFormValues,
+    csrfToken,
+  });
```

Comment:
Do we need to have the CSRF in the `useState()`? It's never going to change, so I think it would be better to just pass it in `dispatch` call.

Reply:
Updated to pass in dispatch